### PR TITLE
Add space in cascade SQL and fix docs.

### PR DESCRIPTION
**Pull request by friedbrice**
Before submitting your PR, check that you've:

- [x] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
- [x] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock

After submitting your PR:

- [x] Update the Changelog.md file with a link to your PR
- [x] Bumped the version number if there isn't an `(unreleased)` on the Changelog
- [x] Check that CI passes (or if it fails, for reasons unrelated to your change, like CI timeouts)

### CODE REVIEW

File: persistent/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog for persistent
 
+## Unreleased
```
Comment (parsonsmatt) [inline persistent/ChangeLog.md:3]:
```suggestion
## 2.11.0.1
```

I'll get this out asap

**Review by parsonsmatt — APPROVED** [review]

**Comment by friedbrice** [comment]
Good to go. Thanks @parsonsmatt!

**Comment by friedbrice** [comment]
Hi, @parsonsmatt. I noticed `persistent-2.11.0.1` made it to Hackage successfully, but could you please also upload `persistent-postgresql-2.11.0.1` to Hackage as well? Thanks!

**Comment by parsonsmatt** [comment]
There weren't any changes made to `persistent-postgresql` in this PR, so I reverted the version bump.

**Comment by friedbrice** [comment]
Oh! I'm so sorry. I was just confused then. I thought the file `ON DELETE` -> ` ON DELETE` was in a file in the postgresql package. Thanks much!

**Pull request merged**